src/screencast/input.ts

```diff
@@ -52,6 +52,13 @@ export class ScreencastInputHandler {
             keyboardEvent.preventDefault();
             keyboardEvent.stopPropagation();
         }
+        if (keyboardEvent.ctrlKey && (keyboardEvent.key === 'c' || keyboardEvent.key === 'x') && keyboardEvent.type === 'keydown') {
```

Comment:
We'll need to handle `metaKey` too for this to work as expected on Macs (not sure if we'll need to make this conditional based on platform).
```suggestion
        if ((keyboardEvent.ctrlKey || keyboardEvent.metaKey) && (keyboardEvent.key === 'c' || keyboardEvent.key === 'x') && keyboardEvent.type === 'keydown') {
```